maps_dashboards/public/components/layer_control_panel/layer_control_panel.tsx

```diff
@@ -63,13 +87,21 @@ const LayerControlPanel = ({ maplibreRef }: Props) => {
     },
   };
 
-  const [layers, setLayers] = useState<ILayerConfig[]>([initialLoadLayer]);
-
+  // Initially load the layers from the saved object
   useEffect(() => {
     maplibreRef.current?.on('load', function () {
-      layers.forEach((layer) => {
-        layersFunctionMap[layer.type]?.initial(maplibreRef, layer);
-      });
+      if (!!mapIdFromUrl) {
```

Comment:
Just confirming, is !! expected?

Reply:
`!!` is used to convert non-Boolean type values to Boolean type